bin/convection

```diff
@@ -178,6 +186,23 @@ module Convection
         end
       end
 
+      def describe_vpc_subnets(format)
+        vpc_template = @cloud.stacks['vpc'].instance_variable_get(:@current_template)
+        raise 'No template defined for [vpc] stack' if vpc_template.nil?
+
+        subnet_map = {}
+        vpc_template['Resources'].select { |_name, attrs| attrs['Type'] == 'AWS::EC2::Subnet' }.each do |name, attrs|
+          subnet_map[attrs['Properties']['AvailabilityZone']] ||= []
+          subnet_map[attrs['Properties']['AvailabilityZone']] << { name: name, cidr: attrs['Properties']['CidrBlock'] }
```

Comment:
1. I think `:subnet_id` be very useful.
2. I'd prefer seeing a structure that lists the subnets, if we call this `describe-subnets` I wouldn't expect availability zone keys (would expect that as a property of each subnet).


I imagined a format similar to the following:

```json
{
  "subnets": [
    {
      "name": "TestSubnet0",
      "subnet_id": "subnet-deadb33f",
      "availability_zone": "us-east-1a",
      "cidr": "10.0.0.0/27"
    },
    {
      "name": "TestSubnet1",
      "subnet_id": "subnet-d34db33f",
      "availability_zone": "us-east-1e",
      "cidr": "10.0.0.192/27"
    }
  ]
}
```